daprdocs/content/en/reference/api/scheduler_api.md

```diff
@@ -0,0 +1,109 @@
+---
+type: docs
+title: "Scheduler API reference"
+linkTitle: "Scheduler API"
+description: "Detailed documentation on the scheduler API"
+weight: 1300
+---
+
+{{% alert title="Note" color="primary" %}}
+Scheduler is currently in alpha.
+{{% /alert %}}
+
+With the Scheduler API, you can orchestrate jobs and tasks in your environment.
+
+## Schedule a job
+
+Schedule a job with the given name.
+
+```
+POST http://localhost:3500/v1.0-alpha1/job/schedule/<name>
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`name` | Job name
+
+### Request content
+
+Any request content will be passed to the scheduler as input. The Dapr API passes the content as-is without attempting to interpret it.
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Request formatted correctly, error in dapr code or underlying component
```

Comment:
```suggestion
`500`  | Request formatted correctly, error in dapr code or Scheduler control plane service
```
The etcd in scheduler isn't a dapr component, maybe we can just say scheduler to encapsulate both scheduler + etcd?